examples/storage_volume_attachments.py

```diff
@@ -0,0 +1,112 @@
+# -*- coding: utf-8 -*-
+###
+# (C) Copyright (2012-2016) Hewlett Packard Enterprise Development LP
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+###
+
+from pprint import pprint
+from hpOneView.oneview_client import OneViewClient
+from hpOneView.exceptions import HPOneViewException
+from config_loader import try_load_from_file
+
+config = {
+    "ip": "172.16.102.59",
+    "credentials": {
+        "userName": "administrator",
+        "password": ""
+    }
+}
+
+# Try load config from a file (if there is a config file)
+config = try_load_from_file(config)
+
+oneview_client = OneViewClient(config)
+
+# Get all volume attachments
+print("Get all volume attachments")
+volume_attachments = oneview_client.storage_volume_attachments.get_all()
+for attachment in volume_attachments:
+    print("   'hostName': '{hostName}' at uri: '{uri}'".format(**attachment))
+
+# Get all storage volume attachments filtering by storage volume URI
+try:
+    print("Get all storage volume attachments filtering by storage volume URI")
+    filter = "storageVolumeUri='/rest/storage-volumes/2E384002-1CB8-43AB-8456-B2239D53BED8'"
+    volume_attachments_filtered = oneview_client.storage_volume_attachments.get_all(filter=filter)
+    for attachment in volume_attachments_filtered:
+        print("   'hostName': '{hostName}' at uri: '{uri}'".format(**attachment))
+except HPOneViewException as e:
+    print(e.msg['message'])
+
+# Get the list of extra unmanaged storage volumes
+print("Get the list of extra unmanaged storage volumes")
+unmanaged_storage_volumes = oneview_client.storage_volume_attachments.get_extra_unmanaged_storage_volumes()
+pprint(unmanaged_storage_volumes)
+
+# Removes extra presentations from a specified server profile.
+try:
+    serverProfileUri = "/rest/server-profiles/1409ac16-8164-4ffa-bfc0-4e44a4f37b51"
```

Comment:
I suggest move any hardcoded ID or URI to the top of the file (near line 40), replacing the value to an empty string and also add a comment indicating that the user must set a value to run the test.

The code must be clear, so in the case you are exemplifying something that has not default values like ID or IP address.